plugins/inputs/all/all.go

```diff
@@ -1 +1,225 @@
+	_ "github.com/influxdata/telegraf/plugins/inputs/salesforce"
+	_ "github.com/influxdata/telegraf/plugins/inputs/sensors"
+	_ "github.com/influxdata/telegraf/plugins/inputs/sflow"
+	_ "github.com/influxdata/telegraf/plugins/inputs/slab"
+	_ "github.com/influxdata/telegraf/plugins/inputs/smart"
+	_ "github.com/influxdata/telegraf/plugins/inputs/snmp"
+	_ "github.com/influxdata/telegraf/plugins/inputs/snmp_legacy"
+	_ "github.com/influxdata/telegraf/plugins/inputs/snmp_trap"
+	_ "github.com/influxdata/telegraf/plugins/inputs/socket_listener"
+	_ "github.com/influxdata/telegraf/plugins/inputs/socketstat"
+	_ "github.com/influxdata/telegraf/plugins/inputs/solr"
+	_ "github.com/influxdata/telegraf/plugins/inputs/sql"
+	_ "github.com/influxdata/telegraf/plugins/inputs/sqlserver"
+	_ "github.com/influxdata/telegraf/plugins/inputs/stackdriver"
+	_ "github.com/influxdata/telegraf/plugins/inputs/statsd"
+	_ "github.com/influxdata/telegraf/plugins/inputs/supervisor"
+	_ "github.com/influxdata/telegraf/plugins/inputs/suricata"
+	_ "github.com/influxdata/telegraf/plugins/inputs/swap"
+	_ "github.com/influxdata/telegraf/plugins/inputs/synproxy"
+	_ "github.com/influxdata/telegraf/plugins/inputs/syslog"
+	_ "github.com/influxdata/telegraf/plugins/inputs/sysstat"
+	_ "github.com/influxdata/telegraf/plugins/inputs/system"
+	_ "github.com/influxdata/telegraf/plugins/inputs/systemd_units"
+	_ "github.com/influxdata/telegraf/plugins/inputs/tail"
+	_ "github.com/influxdata/telegraf/plugins/inputs/tcp_listener"
+	_ "github.com/influxdata/telegraf/plugins/inputs/teamspeak"
+	_ "github.com/influxdata/telegraf/plugins/inputs/temp"
+	_ "github.com/influxdata/telegraf/plugins/inputs/tengine"
+	_ "github.com/influxdata/telegraf/plugins/inputs/tomcat"
+	_ "github.com/influxdata/telegraf/plugins/inputs/trig"
+	_ "github.com/influxdata/telegraf/plugins/inputs/twemproxy"
+	_ "github.com/influxdata/telegraf/plugins/inputs/udp_listener"
+	_ "github.com/influxdata/telegraf/plugins/inputs/unbound"
+	_ "github.com/influxdata/telegraf/plugins/inputs/upsd"
+	_ "github.com/influxdata/telegraf/plugins/inputs/uwsgi"
+	_ "github.com/influxdata/telegraf/plugins/inputs/varnish"
+	_ "github.com/influxdata/telegraf/plugins/inputs/vault"
+	_ "github.com/influxdata/telegraf/plugins/inputs/vsphere"
+	_ "github.com/influxdata/telegraf/plugins/inputs/webhooks"
+	_ "github.com/influxdata/telegraf/plugins/inputs/win_eventlog"
+	_ "github.com/influxdata/telegraf/plugins/inputs/win_perf_counters"
+	_ "github.com/influxdata/telegraf/plugins/inputs/win_services"
+	_ "github.com/influxdata/telegraf/plugins/inputs/wireguard"
+	_ "github.com/influxdata/telegraf/plugins/inputs/wireless"
+	_ "github.com/influxdata/telegraf/plugins/inputs/x509_cert"
+	_ "github.com/influxdata/telegraf/plugins/inputs/xtremio"
+	_ "github.com/influxdata/telegraf/plugins/inputs/zfs"
+	_ "github.com/influxdata/telegraf/plugins/inputs/zipkin"
+	_ "github.com/influxdata/telegraf/plugins/inputs/zookeeper"
+)
```

Comment:
We changed the way we import plugins to be able to selectively build plugins. So can you please rename this file to `azure_monitor.go` and change the content to
```suggestion
//go:build !custom || inputs || inputs.azure_monitor
package all

import _ "github.com/influxdata/telegraf/plugins/inputs/azure_monitor" // register plugin
```